docs/getting-started/install.md

```diff
@@ -195,25 +195,43 @@ cd ~/src
 
 Download and install `libsecp256k1`:
 
+>[!CAUTION]
+>
+>Make sure that `libsecp256k1` version matches flake `libsecp256k1` input version in `iohkNix`: https://github.com/input-output-hk/iohk-nix/blob/master/flake.nix#L14.
+>You can obtain the correct version using this snippet:
+>```bash
+>SECP256K1_VERSION=$(curl https://raw.githubusercontent.com/input-output-hk/iohk-nix/master/flake.lock | jq -r '.nodes.secp256k1.original.ref')
```

Comment:
This seems a bit ugly. How do we maintain historical view with this (eg: 6 months in future ... if reference in iohk-nix changed with node 9.1.0, and one want to compile node-9.0.0 which might need older version)?
Cant this instead be part of node repo itself to be able to scan versions in a way that is consistent with node itself?

Reply:
I see. In general `flake.lock` is a versioned in repo. We can't pin blst version there, as we need the same version across IOG projects. `iohk-nix` is a right place to do so.

In that case, a more precise recipe to obtain the version number would be:
>Get node version from https://github.com/IntersectMBO/cardano-node/tags or use `master` if you'd like to use latest node and use it in `NODE_VERSION`:
>```bash
>NODE_VERSION='8.7.3'
>IOHKNIX_VERSION=$(curl https://raw.githubusercontent.com/IntersectMBO/cardano-node/$NODE_VERSION/flake.lock | jq -r '.nodes.iohkNix.locked.rev')
>SECP256K1_VERSION=$(curl https://raw.githubusercontent.com/input-output-hk/iohk-nix/$IOHKNIX_VERSION/flake.lock | jq -r '.nodes.secp256k1.original.ref')
>echo "Using secp256k1 version: ${SECP256K1_VERSION}"
>```
Would that be better?